utils/cmd_helper.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+from pathlib import Path
+
+
+def clone_repo(repo_url: str, revision: str = None):
+    import subprocess  # nosec - disable B404:import-subprocess check
```

Comment:
why subprocess import inside function while other libs outside?

it is part of standard lib, we usually moving imports inside only for some optional dependencies to prevent requirements to install it if some different function used that does not require this lib